src/IIIFPresentation/BackgroundHandler/appsettings.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Serilog": {
+    "Using": [
+      "Serilog.Sinks.Console"
+    ],
+    "MinimumLevel": {
+      "Default": "Debug",
+      "Override": {
+        "Microsoft": "Warning",
+        "System": "Warning"
+      }
+    },
+    "WriteTo": [
+      {
+        "Name": "Console",
+        "Args": {
+          "outputTemplate": "[{Timestamp:HH:mm:ss} {Level:u3} {SourceContext} {CorrelationId}] {Message:lj}{NewLine}{Exception}"
```

Comment:
Will we need `{CorrelationId}` here? If so, how will it be set? 

An approach I've found useful elsewhere for projects like this (multiple background services) is to use an alternative property name, say, `{ServiceName}`. Then in each listener/handler can push a `ServiceName` property to the current log-context, that way all of the logs contain which service context it's running as. To aid setting `ServiceName` I've used an static method, e.g.

```cs
public static class LogContextHelpers
{
    /// <summary>
    /// Add "ServiceName" property to log context, which is then output as part of default log template.
    /// This is useful to filter logs as there can multiple processes running at the same time 
    /// </summary>
    public static IDisposable SetServiceName(string serviceName) =>
        LogContext.PushProperty("ServiceName", serviceName, false);
}
```

Then `CustomerCreatedMessageHandler` can start with

```cs
public async Task<bool> HandleMessage(QueueMessage message, CancellationToken cancellationToken)
{
    try
    {
        using (LogContextHelpers.SetServiceName(typeof(CustomerCreatedMessageHandler)))
        {
            var customerCreatedMessage = DeserializeMessage(message);
            .. snip ..
```